src/app/core/services/disk-state/disk-state.service.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { createServiceFactory, SpectatorService, mockProvider } from '@ngneat/spectator/jest';
+import { Subject } from 'rxjs';
+import { CoreService } from 'app/core/services/core.service';
+import { CoreEvent } from 'app/interfaces/events';
+import { WebSocketService } from 'app/services/ws.service';
+import { DiskStateService } from './disk-state.service';
+
+describe('DiskStateService', () => {
+  let spectator: SpectatorService<DiskStateService>;
+  const messageBus = new Subject();
+  const fakeSocket = new Subject();
+
+  /*
+   * Mocked Services
+   * */
+  const coreService = mockProvider(CoreService, {
+    register: () => messageBus,
+    emit: (evt: CoreEvent) => messageBus.next(evt),
+  });
+
+  const websocketService = mockProvider(WebSocketService, {
+    sub: () => fakeSocket,
+  });
+
+  const createService = createServiceFactory({
+    service: DiskStateService,
+    providers: [coreService, websocketService],
+    entryComponents: [],
+  });
+
+  /*
+   * Test Methods
+   * */
+
+  beforeEach(() => {
+    spectator = createService();
+  });
+
+  it('should instantiate', () => {
+    expect(spectator).toBeTruthy();
+  });
```

Comment:
This block is probably unnecessary. I think createService() throws if something is not right.